addon/services/context-menu.js

```diff
@@ -5,21 +5,33 @@ import get from 'ember-metal/get';
 import set from 'ember-metal/set';
 import $   from 'jquery';
 
+const itemHeight    = 32;
```

Comment:
You know the itemHeight can be changed with css? So you never know how height it will be?

Reply:
Yes, I'm aware of that, but for now this works for the default either way. I think we need to think about how we can improve this. Maybe it must be able to customize this setting from outside the addon.